manifests/params.pp

```diff
@@ -21,6 +21,11 @@
     default => $::snmp_agentaddress,
   }
 
+  $master = $::snmp_master ? {
+    undef   => 'agentx',
+    default => $::snmp_master,
```

Comment:
Shouldn't this be a boolean to turn it on or off?  `master agentx` is the only valid configuration not `master foo`.